lib/modules/userTagger.js

```diff
@@ -361,7 +361,9 @@ modules['userTagger'] = {
 			this.applyTags();
 			//console.log('after applytags: ' + Date());
 			if (RESUtils.pageType() === 'comments') {
-				RESUtils.watchForElement('newComments', modules['userTagger'].attachVoteHandlers);
+				RESUtils.watchForElement('newComments', function(ele) {
+					modules['userTagger'].attachVoteHandlers(ele.parentNode);
```

Comment:
It looks like, if comments with replies get loaded, then `.attachVoteHandlers()` will affect the replies more than once -- because `ele.parentNode` contains both a single new comment and all its replies.

Lessee, how should we work around this?